sources/reference/tutorials/batch.md

```diff
@@ -226,33 +252,85 @@ latest: digest: sha256:9702b9cf63a6e4961689a661340fc0573d28d0e7f506b90fa5d080e4e
 To actually run your Jobs, Batch needs a Compute environment configured. This is
 where you tell AWS _what_ runs the jobs (i.e EC2 instances/Fargate/etc.).
 
-* Navigate to the Batch console and select `Compute environments`
-* Select `Create`
-* Leave `Managed` selected and provide a name
-    * For the purpose of this guide we'll use `ctk-batch-comp-env`
-* Leave `Fargate` selected under `Instance configuration` but set `Maximum vCPUs`
-to `1`
-* If you want to use a specific VPC, Subnets, and Security Group, select those
-in `Networking`
-    * For this guide, we'll use the values AWS filled in
-* Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
-* Select `Create compute environment`
-
+=== "AWS Console"
+
+    * Navigate to the Batch console and select `Compute environments`
+    * Select `Create`
+    * Leave `Managed` selected and provide a name
+        * For the purpose of this guide we'll use `ctk-batch-comp-env`
+    * Leave `Fargate` selected under `Instance configuration` but set `Maximum vCPUs`
+    to `1`
+    * If you want to use a specific VPC, Subnets, and Security Group, select those
+    in `Networking`
+        * For this guide, we'll use the values AWS filled in
+    * Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
+    * Select `Create compute environment`
+
+=== "AWS CLI"
+
+    * To create your Compute environment, you'll need to know the subnets you'll
+    provide it, take note of the output for the following command, note down the
+    subnet IDs you want to use, replacing `VPC_ID` with the VPC ID you're deploying
```

Comment:
Is it possible to auto-fill the values that AWS would fill in if we were doing it manually: https://github.com/chaostoolkit/chaostoolkit-documentation/pull/142/files#diff-14961ade072f5656a60b94a37b75aaa8d80bb0ff25da189d429f39cd38e92dbfR265
If not, might we need instructions on how to retrieve that value?

Reply:
Which value do you mean?

If you mean the subnets, that's just what the command below this text is doing, listing the subnets

Reply:
I am meaning the `VPC_ID`
To access the default one, I went to create one manually on AWS and used the id it gave me to replace "VPC_ID"

Reply:
So we could show them how to list VPCs, but for your case, the default VPC is the only vpc available in the account

Reply:
Sounds good and yes I understand that!
I just wasn't sure how I would retrieve the VPC_ID to initiate that command in the first place
Are we just expecting the users to have theirs easily accessible?

Reply:
Big assumption is someone will have the AWS console up/know how to query for this, but I've added a step

Reply:
We have to remember that we're not trying to teach folks AWS, just how to setup CTK on AWS, which admittedly is a fine line

Reply:
Makes sense and looks good!